client/src/app/site/deployment-center/deployment-center-setup/step-configure/configure-vsts-build/linux-frameworks/linux-frameworks.component.ts

```diff
@@ -134,12 +134,19 @@ export class LinuxFramworksComponent implements OnDestroy {
 
   private setupValidators(stack: string) {
     // Regex value comes from Azure Devops team for validation
-    const validator = RegexValidator.create(
+    const nodeValidator = RegexValidator.create(
       new RegExp('^$|^(node|pm2|ng)\\s+\\w+'),
       this._translateService.instant(PortalResources.invalidStartupCommandNodejs)
     );
+    const aspNetCoreValidator = RegexValidator.create(
+      new RegExp('^$|^(dotnet)\\s+\\w+'),
```

Comment:
You should remove the initial ^$. Do the same from above also

Reply:
Done